ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/README.md

```diff
@@ -0,0 +1,180 @@
+:scrollbar:
+:data-uri:
+:toc2:
+:linkattrs:
+
+= 3scale Latest Product Environment
+
+:numbered:
+
+== Overview
+
+This workload provisions all the required resources for 3scale API Manager & a default tenant On OpenShift 4. The following are provisioned:
+
+. 3scale Operator
+. 3scale Multitenant API Manager (default channel) (S3 compatible provider)
+. 3scale single tenant
+. 3scale Gateway Projects for each user:
+.. Gateway Operator (latest channel)
+.. Portal Endpoint secret for connecting to corresponding 3scale tenant admin
+.. Staging & Production API Gateways
+
+
+=== Prerequisites and assumptions
+
+An OCP 4 cluster is provisioned and this role is run using a cluster admin.
+
+
+. The version of 3scale provisioned in this lab (v2.11) is known to run on OpenShift Container Platform v4.*.
++
+This version of OpenShift should already be pre-installed before executing this ansible role.
+
+. Using a version of oc utility that corresponds to your target OCP cluster, ensure oc utility is already authenticated as the cluster-admin.
+
+. This ansible role requires installation of the lxml python module on target host executing this ansible. ie:
++
+-----
+# dnf install python3-lxml
```

Comment:
Is that something that is missing from the bastion VMs? If so the role should install it (using the dnf module).